### PR TITLE
tests: add integration test and ci workflow for real tiup

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Ref #7298. wait https://github.com/tikv/pd/pull/7304 merged

### What is changed and how does it work?

```commit-message
support tests with real tiup
```

### Check List

Tests

- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,7 +46,12 @@ ifeq ($(PLUGIN), 1)
 	BUILD_TAGS += with_plugin
 endif
 
-LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(shell git describe --tags --dirty --always)"
+RELEASE_VERSION ?= $(shell git describe --tags --dirty --always)
+ifeq ($(RUN_CI), 1)
+	RELEASE_VERSION := None
```
Comment (JmPotato) [inline Makefile:51]: Why do we need to set it to `None`?

File: client/http/api.go [hunk 2]
```diff
@@ -29,7 +29,13 @@ const (
 	regionsByKey        = "/pd/api/v1/regions/key"
 	regionsByStoreID    = "/pd/api/v1/regions/store"
 	Stores              = "/pd/api/v1/stores"
+	Store            = "/pd/api/v1/store"
```
Comment (JmPotato) [inline client/http/api.go:32]: Please format it.

**Review by JmPotato — APPROVED** [review]
The rest LGTM. We could gradually improve the test cases later.

File: Makefile (hunk 1 above)
Comment (HuSharp) [reply Makefile:51]: there is no tag in the ci environment, we can not pass https://github.com/tikv/pd/blob/bfc988bb37a738b3bba2e85903d304af5d8532d6/server/util.go#L39-L41  whithout tag :(

**Comment by nolouch** [comment]
need fix conflicts.

**Comment by JmPotato** [comment]
/hold

We are trying to use unified Jenkins CI integration instead of using GitHub Actions.

**Comment by HuSharp** [comment]
replaced by https://github.com/tikv/pd/pull/7493

**Pull request closed without merging**